### PR TITLE
some fixes

### PR DESCRIPTION
**Pull request by OriolAbril**

### CODE REVIEW

File: 2022-07_sprint/schedule.md [hunk 1]
```diff
@@ -4,19 +4,17 @@
...
 :::
 
 ## Schedule with Timezones
 
-<p float="left">
-   <a href="https://docs.google.com/spreadsheets/d/1oO9OhsTh9PFcqKI7Exj_ZcH0GvunQVHtM4aqUh0kjTk/edit?usp=sharing">
-<img alt="schedule with timezones" src="../_static/2022_07/2022_07_timezones.png">
-</a>
-</p>
+[![schedule with timezones](../_static/2022_07/2022_07_timezones.png)](https://docs.google.com/spreadsheets/d/1oO9OhsTh9PFcqKI7Exj_ZcH0GvunQVHtM4aqUh0kjTk/edit?usp=sharing)
```
Comment (OriolAbril) [inline 2022-07_sprint/schedule.md:17]: I think only the image or the embedding should be present, but not both. What do you prefer?

* dropdown vs no dropdown
* image vs embedding

File: 2022-07_sprint/schedule.md [hunk 2]
```diff
@@ -4,19 +4,17 @@
 :::{include} schedule_table.md
 :::
 
+
 :::{dropdown} Open dropdown for spreadsheet with times in more time zones
 :icon: clock
 
 <iframe src="https://docs.google.com/spreadsheets/d/e/2PACX-1vSJ_x4V63aji-WmarsgEyj9eS_Cp0DB5m647TZdNjYj03nPQq0u0Q6eKPGhHrw-8Ugp16KhAg-AgLkK/pubhtml?widget=true&amp;headers=false"  height="320" width="750"></iframe>
+
```
Comment (reshamas) [inline 2022-07_sprint/schedule.md:12]:
```suggestion
```

Comment (reshamas) [reply 2022-07_sprint/schedule.md:12]: @OriolAbril 
I tested both on mobile, and the screenshot image+link is easier to navigate.

**Comment by OriolAbril** [comment]
@reshamas https://github.com/pymc-devs/pymc-data-umbrella/pull/87#discussion_r901122807 does not delete the closing colon fence (`:::`) of the dropdown directive it deletes. Now everything below that is interpreted as code: https://pymc-data-umbrella--87.org.readthedocs.build/en/87/2022-07_sprint/schedule.html

**Comment by reshamas** [comment]
@OriolAbril 
Ok, I will submit another PR to fix this.

**Comment by OriolAbril** [comment]
[Directives](https://sphinx-primer.readthedocs.io/en/latest/core/directives.html) can be defined using backtick ```` ``` ```` or colon `:::` fences and need to be opened and closed.

They can be nested like one would do in markdown code blocks by adding more elements to the more external fences and both closing and opening fences must have the same number of elements in the fence. Issues with non-pairing of fences (either backticks or colon ones) end up with rendering like that or empty code cells lying around.

**Pull request merged**